lib/utils.bash

```diff
@@ -431,7 +437,7 @@ get_plugin_source_url() {
 }
 
 find_tool_versions() {
-  find_file_upwards ".tool-versions"
```

Comment:
We don't necessarily want to just find `.tool-versions` files, so we use the helper here.